CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
+- Added setting to ensure YouTube links always load in brave. ([#8272](https://github.com/brave/brave-ios/issues/8272))
+- Added confirmation message for successfully exporting bookmarks. ([#8236](https://github.com/brave/brave-ios/issues/8236))
+- Added confirmation message for resetting configuration in VPN settings. ([#8087](https://github.com/brave/brave-ios/issues/8087))
+- Added support for additional schemas for going to the App Store. ([#8035](https://github.com/brave/brave-ios/issues/8035))
+- Added VPN improvements for callouts and the settings page. ([#4879](https://github.com/brave/brave-ios/issues/4879))
+- Added the ability to download .mobileconfig files and auto-import into settings app. ([#4358](https://github.com/brave/brave-ios/issues/4358))
+- Implemented tabs bar re-order via drag and drop. ([#8058](https://github.com/brave/brave-ios/issues/8058))
+- Removed SDWebImage SVG coder. ([#7026](https://github.com/brave/brave-ios/issues/7026))
+- Removed cookie consent info popup. ([#8217](https://github.com/brave/brave-ios/issues/8217))
+- Removed the dislike swipe action for ad notifications. ([#8211](https://github.com/brave/brave-ios/issues/8211))
+- Updated the “Send” button in “Send to your Devices” window to be displayed in the header section. ([#8329](https://github.com/brave/brave-ios/issues/8329))
+- Updated filter lists to increase the amount of engines that can be enabled on low memory devices. ([#8292](https://github.com/brave/brave-ios/issues/8292))
+- Updated the “Report a Broken Site” submission form to include details and contact fields. ([#8270](https://github.com/brave/brave-ios/issues/8270))
+- Updated user agent for iOS 17. ([#8155](https://github.com/brave/brave-ios/issues/8155))
+- Updated browser colors. ([#8078](https://github.com/brave/brave-ios/issues/8078))
+- Updated Search Settings page to remove “Edit” option when custom search engines are not added. ([#5987](https://github.com/brave/brave-ios/issues/5987))
+- Fixed toolbar being displayed in landscape mode when changing orientation. ([#8421](https://github.com/brave/brave-ios/issues/8421))
+- Fixed external links not working on New Tab Page. ([#8383](https://github.com/brave/brave-ios/issues/8383))
+- Fixed content blocking to be removed after disabling Ad-blocking. ([#8369](https://github.com/brave/brave-ios/issues/8369))
+- Fixed issue with pages being rendering poorly on https://youtube.com. ([#8364](https://github.com/brave/brave-ios/issues/8364))
+- Fixed tabs being cleared after relaunching browser. ([#8362](https://github.com/brave/brave-ios/issues/8362))
+- Fixed VPN status to be updated in the menu without having to open the settings page. ([#8317](https://github.com/brave/brave-ios/issues/8317))
+- Fixed context menu item when using long press action on Favorites. ([#8286](https://github.com/brave/brave-ios/issues/8286))
```

Comment:
Lets mention what was exactly fixed as the it's pretty vague right now. For example:

```
- Fixed "Edit Favorite" appearing as "Edit Bookmark" via context menu when using long press action on Favorites.
```